.evergreen/build-manylinux.sh

```diff
@@ -6,24 +6,20 @@ docker version
 # https://github.com/docker/setup-qemu-action/blob/2b82ce82d56a2a04d2637cd93a637ae1b359c0a7/README.md?plain=1#L46
 docker run --rm --privileged tonistiigi/binfmt:latest --install all
 
-# manylinux1 2021-05-05-b64d921 and manylinux2014 2021-05-05-1ac6ef3 were
+# manylinux2014 2021-05-05-1ac6ef3 was
 # the last releases to generate pip < 20.3 compatible wheels. After that
 # auditwheel was upgraded to v4 which produces PEP 600 manylinux_x_y wheels
 # which requires pip >= 20.3. We use the older docker image to support older
 # pip versions.
 BUILD_WITH_TAG="$1"
 if [ -n "$BUILD_WITH_TAG" ]; then
-  images=(quay.io/pypa/manylinux1_x86_64:2021-05-05-b64d921 \
-          quay.io/pypa/manylinux1_i686:2021-05-05-b64d921 \
-          quay.io/pypa/manylinux2014_x86_64:2021-05-05-1ac6ef3 \
+  images=(quay.io/pypa/manylinux2014_x86_64:2021-05-05-1ac6ef3 \
```

Comment:
Is the removal of i686 unrelated?

Reply:
If I understand correctly, manylinux1 wheels are Centos5. Is that what we were using before, and this upgrades to Centos7? We can't move to manylinux_x_y because it requires the new pip solver. Please respond if I've gotten any of this wrong.

I think I understand what's going on and why. Looks good!

Reply:
Both x86 and i686 were manylinux1, but we're going to try and bring both back.  Yes, this is related so supporting older versions of pip that come preinstalled on stock Centos 7.  See the compat table here: https://github.com/pypa/manylinux.  If we bump to manylinux2014, then we'd still support CentOS 7 rh-python38, but not stock CentOS 7.